iOS/Shimmer/nuget/Xamarin.Shimmer.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.Shimmer</id>
+    <title>Shimmer for Xamarin.iOS</title>
+    <version>1.0.2</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Shimmer is an easy way to add a shimmering effect to any view in your app.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://components.xamarin.com/view/shimmer</projectUrl>
+    <licenseUrl>https://components.xamarin.com/license/shimmer</licenseUrl>
```

Comment:
This should be a FWLink as detail here https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15